test-non-capturing-groups/test-regex.lpi

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we still need this separate test project?
Now we have the test cases in main test suit

Reply:
Removed it.